pages/common/adb-shell-wm.md

```diff
@@ -0,0 +1,12 @@
+# adb shell wm
+
+> Show information about the screen of an Android device.
+> More information: <https://adbinstaller.com/commands/adb-shell-wm-5b672b17e7958178a2955538>.
```

Comment:
I wonder if we should use something more generic like this, because the current URL contains a hash, which worries me that it might change (frequently.)
```suggestion
> More information: <https://adbinstaller.com/commands>.
```
Besides, https://adbinstaller.com/commands is _very_ obvious as to what's going on.

If you do apply this, also apply it to the other `adb` commands you've made (and maybe to the other ones already existing 🤔 )